Documented path option for data prepper http sink (#9593)
opensearch-project/documentation-website: _data-prepper/pipelines/configuration/sources/http.md

Option | Required | Type | Description
:--- | :--- | :--- | :---
port | No | Integer | The port that the source is running on. Default value is `2021`. Valid options are between `0` and `65535`.

> path | No | String | The  URI path for log ingestion and it should start with `/`. Path can contain `${pipelineName}` placeholder which will be replaced with pipeline name. Default value is `/log/ingest`. Path example `/${pipelineName}/logs`

Reviewer: ```suggestion
path | No | String | The URI path for log ingestion should start with a forward slash (/), for example `/${pipelineName}/logs.` The ${pipelineName} placeholder will be replaced with the pipeline name. The default value is `/log/ingest`.
```
